File: kore_api/kore_api/koredb/Account.cs
using System;
using System.Collections.Generic;

namespace kore_api.koredb
{
    public partial class Account
    {
        public Account()
        {
            Task = new HashSet<Task>();
            Taskmembership = new HashSet<Taskmembership>();
        }

        public int Id { get; set; }
        public int OrgId { get; set; }
        public string AccountName { get; set; }
        public DateTime DateCreated { get; set; }
        public DateTime DateModified { get; set; }
        public int? Status { get; set; }
        public string Description { get; set; }
        public int? CreatedBy { get; set; }
        public int? ModifiedBy { get; set; }

        public Organization Org { get; set; }
        public ICollection<Task> Task { get; set; }
        public ICollection<Taskmembership> Taskmembership { get; set; }
    }
}

File: kore_app/lib/models/account.dart
class Account {
  int id;
  String name;
  String dateCreated;
  String dateModified;
  int status;
  String description;
  int createdBy;
  int modifiedBy;
  double percentage;

  Account({
    this.id,
    this.name, 
    this.dateCreated,
    this.dateModified,
    this.status,
    this.description,
    this.createdBy,
    this.modifiedBy,
    this.percentage
  });

  factory Account.fromJson(Map<String, dynamic> json ) {
    return new Account(
        id: json['id'] as int,
        name: json['accountName'] as String,
        dateCreated: json['dateCreated'] as String,
        dateModified: json['dateModified'] as String,
        status: json['status'] as int,
        description: json['description'] as String,
        createdBy: json['createdBy'] as int,
        modifiedBy: json['modifiedBy'] as int,
        percentage: 100.0
    );
  }
}